web/src/interfaces.ts

```diff
@@ -2,3 +2,11 @@ export interface Image {
   url: string;
   alt?: string;
 }
+
+export interface Customer {
+  id: number;
+  gpayId: string;
```

Comment:
@ifhuang This is the gpay id of the Customer. the get customer endpoint returns this.

Reply:
ic. which spot api/field provides this id?

Reply:
@ifhuang The identity api! https://developers.google.com/pay/spot/eap/reference/identity-api Will be using the `sub` since they says that's the identifier